docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
     image: lmenezes/cerebro:0.9.3
     container_name: pfi-cerebro
     ports:
-      - 9091:9000
+      - 9092:9000
```

Comment:
what's the motivation for changing this?

Reply:
I needed to get minio port forwarding working so I could inspect local S3. For that I used 9091 since minio already exposes 9090.

This meant I had to bump this port number.

This change could've done with its own PR - normally I go the extra mile to prise apart the different concerns that have accumulated on a branch but this task is dragging on a bit so I didn't bother this time.

Reply:
👍🏻 makes sense!